### ISSUE TITLE
BUG: hud_editor adds one extra bottom row to the items edit green box.

### ISSUE THREAD

**Issue by Matrix-QW** [ezQuake/ezquake-source#536]
**ezQuake version:**
ezQuake 3.6-dev-alpha7 r7034~0df39021 Mar  7 2021, 20:06:58

**Describe the bug**
What the subject says but the items' pixel placement on screen is correct.
It's just confusing trying to place items guided by the bottom green line because it overlaps with the upper line of another item placed below it and also if an item is set at the bottom of the screen the line goes off screen.
This doesn't happen with the latest ezQuake 3.2.3.

**To Reproduce**
1. hud_item_align_x "center"
2. hud_item_align_y "bottom"
3. hud_item_opacity 1
4. hud_item_place "screen"
5. hud_item_pos_x 0
6. hud_item_pos_y 0
7. hud_item_show 1

**Expected behavior**
The bottom line of the box must match pixel placement.
